src/components/Register/index.jsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import { Link } from 'react-router-dom';
+import Grid from '@material-ui/core/Grid';
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Snackbar from 'components/Snackbar'
+import { connect } from 'react-redux'
+import { push } from "connected-react-router";
+
+import TextInput from 'components/TextInput';
+import useRegisterForm from 'hooks/useRegisterForm';
+
+const useStyles = makeStyles(theme => ({
+  '@global': {
+    body: {
+      backgroundColor: theme.palette.common.white,
+    },
+  },
+  paper: {
+    marginTop: theme.spacing(8),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  avatar: {
+    margin: theme.spacing(1),
+    backgroundColor: theme.palette.secondary.main,
+  },
+  form: {
+    width: '100%', // Fix IE 11 issue.
+    marginTop: theme.spacing(3),
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+}));
+
+const Register = ({push}) => {
+  const classes = useStyles();
+  const { inputs, handleInputChange, handleSubmit } = useRegisterForm();
+  const [open, setOpen] = React.useState(false)
+
+
+  const handleSubmitWrapper = async (event) => {
+    const result = await handleSubmit(event)
+    if (result && result.success) {
+        setOpen(prev => !prev)
+        setTimeout(() =>{
+          push('/login')
+        }, 1000)
+    }
+  }
+
+  return (
+    <Container component="main" maxWidth="xs">
+      <Snackbar variant="success" message="Registration Successful. Redirecting to Sign In Page" isOpen={open}/>
```

Comment:
Not huge, but if a user registers, do you think they should already be signed in rather than having to sign in right afterwards? What do you think?